spec/requests/items_requests_spec.rb

```diff
@@ -134,6 +135,49 @@
       end
     end
 
+    describe "with invalid parameters" do
+      let(:invalid_item_params) do
+        {
+          item: {
+            name: "",  # Invalid name
+            partner_key: create(:base_item).partner_key,
+            value_in_cents: -100,  # Invalid value
+            package_size: nil,
+            distribution_quantity: nil
+          }
+        }
+      end
+
+      let(:categories) do
+        [
+          FactoryBot.create(:item_category, name: 'Apples', organization: organization),
+          FactoryBot.create(:item_category, name: 'Umbrella', organization: organization),
+          FactoryBot.create(:item_category, name: 'Bananas', organization: organization)
+        ]
+      end
+    
+      before do
+        categories  # Ensure categories are created before the request
+      end
```

Comment:
You can remove this `before` block and replace the `let` with `let!` to ensure they are created.